packages/terra-form-checkbox/themes/clinical-lowlight-theme/clinical-lowlight-theme.scss

```diff
@@ -0,0 +1,137 @@
+  --terra-form-checkbox-checked-before-symbol-padding-left: 0;
+  --terra-form-checkbox-checked-before-symbol-padding-right: 0;
+  --terra-form-checkbox-checked-before-symbol-padding-top: 0;
+  --terra-form-checkbox-checked-before-width: 1rem;
+  --terra-form-checkbox-checked-disabled-before-border-width: 1px;
+
+  --terra-form-checkbox-is-mobile-before-height: 1rem;
+  --terra-form-checkbox-is-mobile-before-margin-top: 0.142857rem;
+  --terra-form-checkbox-is-mobile-before-width: 1rem;
+
+  --terra-form-checkbox-is-mobile-checked-before-height: 1rem;
+  --terra-form-checkbox-is-mobile-checked-before-line-height: 1rem;
+  --terra-form-checkbox-is-mobile-checked-before-margin-top: 0.142857rem;
+  --terra-form-checkbox-is-mobile-checked-before-width: 1rem;
+
+  --terra-form-checkbox-hover-before-border-color: #585a5c;
+  --terra-form-checkbox-hover-before-border-width: 1px;
+  --terra-form-checkbox-hover-checked-before-border-color: #005887;
+  --terra-form-checkbox-hover-checked-before-border-width: 1px;
+
+  --terra-form-checkbox-disabled-before-font-color: rgba(212, 222, 225, 0.3);
+  --terra-form-checkbox-disabled-before-font-opacity: 1;
+  --terra-form-checkbox-disabled-before-background-color: rgba(34, 42, 46, 0.5);
+  --terra-form-checkbox-disabled-before-background-image: none;
+  --terra-form-checkbox-disabled-before-border-color: rgba(178, 181, 182, 0.2);
+  --terra-form-checkbox-disabled-before-border-width: 1px;
+
+  --terra-form-checkbox-focus-before-border-color: #004c76;
+  --terra-form-checkbox-focus-before-border-width: 1px;
+
+  --terra-form-checkbox-focus-after-ring-background-color: rgba(0, 76, 118, 0.35);
+  --terra-form-checkbox-focus-after-ring-border-radius: 4px;
+  --terra-form-checkbox-focus-after-ring-height: 1.42857rem;
+  --terra-form-checkbox-focus-after-ring-left: -0.21428rem;
+  --terra-form-checkbox-focus-after-ring-top: -0.071428rem;
+  --terra-form-checkbox-focus-after-ring-width: 1.42857rem;
+
+  @include terra-inline-svg-var('--terra-form-checkbox-checked-disabled-before-content', '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="rgba(178,181,182,0.2)" d="M18.4,40.4L0,22l4.2-4.2L18.4,32L43.8,6.6l4.2,4.2L18.4,40.4z"/></svg>');
+  @include terra-inline-svg-var('--terra-form-checkbox-checked-before-content', '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#002b42" d="M12.2,44.5L0,32.3v-4.9V22l18.4,18.4L48,10.8l0,5.4l0,4.9l-24,24C20.9,48,16,48.6,12.2,44.5z"/><path fill="#b2b5b6" d="M18.4,40.4L0,22l4.2-4.2L18.4,32L43.8,6.6l4.2,4.2L18.4,40.4z"/></svg>');
+  @include terra-inline-svg-var('--terra-form-checkbox-field-error-background', '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><path fill="#fb4c4c" d="M24 0c13.3 0 24 10.7 24 24S37.3 48 24 48 0 37.3 0 24C0 10.8 10.7 0 23.9 0h.1z"/><path fill="#FFF" d="M7 21.5h34v5H7v-5z"/></svg>');
+
+  // -------------------- Version 2.x -------------------- //
```

Comment:
Are we intended to maintain these versions?

Reply:
No, we will not be maintaining the obsolete version as mentioned here https://github.com/cerner/terra-core/pull/2758#issuecomment-574224255.
This has been addressed as part of commit https://github.com/cerner/terra-core/pull/2758/commits/28a7c8dc409e0e443329574f2c3b8e220c7fa250